serverless.yaml

```diff
@@ -96,6 +100,18 @@ provider:
       Effect: Allow
       Resource:
         - '*'
+    - Action:
+        - 's3:ListBucket'
+        - 's3:GetObject'
+      Effect: Allow
+      Resource:
+        - !GetAtt BulkExportResultsBucket.Arn
+        - !Join ['', [!GetAtt BulkExportResultsBucket.Arn, '/*']]

```

Comment:
Just for my learning why do we need both here?

Reply:
First one is a Bucket arn. Second one is an Object arn (with a wildcard).
`s3:ListBucket` operates on Buckets.  `s3:GetObject` operates on Objects. Both resources are needed to make the actions valid